### PR TITLE
Book mappings (from #2157)

### PR DESCRIPTION
**Pull request by IotaBread**
Cherry picked changes from #2157, also updated Enigma

### CODE REVIEW

**Comment by IotaBread** [comment]
Looks like I did something wrong

**Comment by liach** [comment]
I suggest you cherry pick the commit from #2171

**Comment by IotaBread** [comment]
> I suggest you cherry pick the commit from #2171

Oh, I just pushed with the ones from #2157. Anyways, it should be fine now

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-enigma_version=0.21.4+build.225
+enigma_version=0.26.0
```
Comment (liach) [inline gradle.properties]:
```suggestion
enigma_version=0.26.1
```

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Comment by YanisBft** [comment]
Ready to merge because everything here has been approved in the snapshot PR

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**